lifting-state-up/single-source-of-truth.md

```diff
@@ -1,6 +1,7 @@
 ## Single Source of Truth
```

Comment:
Just noticed while clicking through that this is h2 rather than an h1. Can we fix this too? Either part of this PR or a follow up is fine.